src/data-sources/plugins/DataSourcePlugin.ts

```diff
@@ -31,7 +31,7 @@ export interface IDataSourcePlugin {
   };
 
   bind (actionClass: any): void;
-  updateDependencies (dependencies: IDictionary, args: IDictionary, callback: () => void): void;
+  _updateDependencies (dependencies: IDictionary, args: IDictionary, callback: () => void): void;
```

Comment:
I believe that our naming convention for class functions don't use underscore prefix "_"